src/webui/static/app/tvheadend.js

```diff
@@ -489,7 +489,7 @@ function accessUpdate(o) {
         var chepg = new Ext.TabPanel({
             activeTab: 0,
             autoScroll: true,
-            title: _('Channel / EPG'),
+            title: _('Channel list'),
             iconCls: 'television',
```

Comment:
The tab includes Channel tags, EPG grabber modules etc. So yes I think you're right. I just didn't think Channels / EPG was correct either.  I suppose my change makes even _LESS_ sense now.